web-js/4.x/graphics-layer-from-rest/README.MD

```diff
@@ -0,0 +1,6 @@
+# HTML In Popups
+
+This sample shows how create a Graphics Layer with external data.
```

Comment:
Could you please add a little more detail about how this sample works? You can reference some of the other 3.x/4.x samples for examples.

The readme should explain which parts of the API are being used to get the resulted outcome (e.g. what is being used to send the request being sent, etc.)